examples/timeline-charts-example/README.md

```diff
@@ -0,0 +1,64 @@
+# MongoDB Charts Embedding Example - Timeline Chart
+
+## Background
+
+📄 _[See the MongoDB Charts Embedding Docs for more details](https://docs.mongodb.com/charts/saas/embedding-charts/)_
+
+The example code in this directory is building a small react app, implementing a chart timeline using the Charts Embedding SDK.
+
+What the application is doing is showing the distribution of Olympic medals through the years.
+
+We are changing the filter on the data every few seconds so that it can look like a timeline video.
+
+The idea is this - for every Olympic year we are filtering all data from the beginning of the Olympic games to the current year.
```

Comment:
```suggestion
We are using a [filter](https://docs.mongodb.com/charts/saas/filter-embedded-charts/#filter-data-on-charts-embedded-with-the-sdk) on the Olympic year, changing the filter every few seconds to turn it into a timelapse.
```

Reply:
You are right! Thanks, I will change it.